test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenTests.cs

```diff
@@ -3,17 +3,35 @@
 
 using System.Collections.Generic;
 using System.Security.Claims;
+using System.Text;
 using Microsoft.IdentityModel.Logging;
 using Microsoft.IdentityModel.TestUtils;
 using Microsoft.IdentityModel.Tokens;
 using Xunit;
 
-#pragma warning disable CS3016 // Arrays as attribute arguments is not CLS-compliant
-
 namespace System.IdentityModel.Tokens.Jwt.Tests
 {
     public class JwtSecurityTokenTests
     {
+        [Fact]
+        public void DateTime2038Issue()
+        {
+            var key = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(new string('a', 128)));
+            var creds = new SigningCredentials(key, SecurityAlgorithms.HmacSha256);
+
+            var claims = new[] { new Claim(ClaimTypes.NameIdentifier, "Bob") };
+            var token = new JwtSecurityToken(
+                issuer: "issuer.contoso.com",
+                audience: "audience.contoso.com",
+                claims: claims,
+                expires: (new DateTime(2038, 1, 20)).ToUniversalTime(),
+                signingCredentials: creds);
+
+            Assert.Equal(token.ValidTo, (new DateTime(2038,1,20)).ToUniversalTime());
+            foreach (var claim in token.Claims)
+                Console.WriteLine($"{claim}");
```

Comment:
this is just going to be noise in the tests isn't it?

Reply:
yeah, i will remove that